sdk/src/main/java/io/verloop/sdk/ui/VerloopFragment.kt

```diff
@@ -314,8 +346,8 @@ class VerloopFragment : Fragment() {
     }
 
     private fun callJavaScript(script: String) {
-        Log.d(TAG, "javascript: $script")
-        mWebView?.evaluateJavascript(script, null)
+        logEvent(LogLevel.DEBUG, "Calling javascript: $script")
```

Comment:
Make a list of constant strings which will be sent as a payload. If there are variables which you are passing, keep it as a json in the third parameter. Eg: [LogLevel.DEBUG, JS_CALL, {script: some_script}]